app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people.update({self.name: self})
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons = []
+
+    for person_data in people:
+        persons.append(Person(person_data.get("name"), person_data.get("age")))
+
+    for person_data in people:
+        person = Person.people.get(person_data.get("name"))
```

Comment:
Using `Person.people.get()` is a good practice to avoid `KeyError`, but ensure that you handle the case where the name might not be found, possibly by checking if `person` is `None`.